script.js

```diff
@@ -0,0 +1,188 @@
+/*var reset = document.getElementById("reset");
+function turnPageRed() {
+  document.body.style.backgroundColor = "red";
+}
+reset.addEventListener("click", turnPageRed);*/
+var submitTally = 0;
+var userWonGameReset = 0;
+
+function xAndBlank(temporaryPlaceholderVariable, quadrant, actualTextSpace) {
+  var temporaryPlaceholderVariable = document.getElementById(quadrant);
```

Comment:
this variable name could be more expressive, I think it's holding a cell, why not call it `var cell`?